### DISCUSSION TITLE
wt32-eth01 cmt2300a

### DISCUSSION THREAD

**Discussion by quarkenbush** [tbnobody/OpenDTU#1241] [Q&A]
Hallo

hat jemand die Pinbelegung für wt32-eth01 cmt2300a

**Comment by bulli12345** [comment]
Bei mir funktioniert:
[
        "name": "CMT2300A_WT32-ETH01",
        "nrf24": {
            "miso": -1,
            "mosi": -1,
            "clk": -1,
            "irq": -1,
            "en": -1,
            "cs": -1
        },
        "cmt": {
            "clk": 32,
            "cs": 15,
            "fcs": 2,
            "sdio": 4,
            "gpio2": -1,
            "gpio3": -1
        },
        "eth": {
            "enabled": true,
            "phy_addr": 1,
            "power": 16,
            "mdc": 23,
            "mdio": 18,
            "type": 0,
            "clk_mode": 0
        }
    }
]
